applications/nrf5340_audio/src/drivers/cs47l63_reg_conf.h

```diff
@@ -31,9 +31,18 @@
 /* clang-format off */
 /* Set up clocks */
 const uint32_t clock_configuration[][2] = {
+#if CONFIG_AUDIO_SAMPLE_RATE_HZ == 16000
+	{ CS47L63_SAMPLE_RATE1, 0x0012},
+#elif CONFIG_AUDIO_SAMPLE_RATE_HZ == 24000
+	{ CS47L63_SAMPLE_RATE1, 0x0002},
+#elif CONFIG_AUDIO_SAMPLE_RATE_HZ == 48000
+	{ CS47L63_SAMPLE_RATE1, 0x0003},
+#else
+	#error "current AUDIO_SAMPLE_RATE_HZ setting not supported"
+#endif
 	{ CS47L63_SYSTEM_CLOCK1, 0x034C },
 	{ CS47L63_ASYNC_CLOCK1, 0x034C },
-	{ CS47L63_FLL1_CONTROL2, 0x88208020 },
+	{ CS47L63_FLL1_CONTROL2, 0x88600008 },
```

Comment:
What does this change do?

Reply:
It changes the source of the FLL in CS47L63, I tried both with the old settings and the new, only the new works when changing to something else than 48kHz, might need to look at this next week. @rick1082 has some more details on why this was chosen

Reply:
We need to double check this before merging this PR

Reply:
I've turned off the phase-detect again and the other settings should be good now. We have to use MCLK (6.144MHz) since the BCLK will be too low if the sample rate is lower than 48kHz.